cps/kni.c

```diff
@@ -884,40 +933,12 @@ kni_cps_rd_event(struct cps_config *cps_conf)
 			break;
 		}
 
-		ret = mnl_cb_run(buf, ret, 0, 0,
-			rd_cb, &updates[num_updates]);
+		ret = mnl_cb_run(buf, ret, 0, 0, rd_cb, NULL);
```

Comment:
Let's pass `cps_conf` as the `arg` of `rd_cb()`, so we can pass `cps_conf` to `rd_modroute()`. This avoids the need to call `get_cps_conf()` in `rd_modroute()`. My intention here is to minimize the use of functions `get_XXX_conf()` because they will make our life harder once we want to create contexts similarly to containers in the Linux kernel.